src/test/deploy/functions/backend.spec.ts

```diff
@@ -629,6 +635,19 @@ describe("Backend", () => {
       expect(backend.someEndpoint(bkend, (fn) => fn.id === "missing")).to.be.false;
     });
 
+    it("removeEndpoints", () => {
+      // Add our endpoint
+      bkend.endpoints[endpointEU.region] = { [endpointREMOVED.id]: endpointREMOVED };
```

Comment:
nit* A more idiomatic way of creating a backend from endpoint is to use `of` fn:

```
const b = backend.of(endpointREMOVED)
```